docs/detections/rules-ui-manage.asciidoc

```diff
@@ -79,20 +77,32 @@ NOTE: For prebuilt rules, you can only activate, deactivate, delete, edit
 
 [float]
 [[import-export-rules-ui]]
-=== Import and export rules
+=== Export and import rules
 
-. Go to *Detect* -> *Rules*.
-. To import rules:
-.. Click *Import rules*.
-.. Drag and drop files that contain the detection rules.
-+
-NOTE: Imported rules must be in an `ndjson` file.
+You can export detection rules to an `.ndjson` file, which you can then import into another {es-sec} environment.
+
+[IMPORTANT]
+=================
+Detection rule actions are included in the exported file, but the connectors used by the actions are not included. 
 
+Use the {kibana-ref}/managing-saved-objects.html#managing-saved-objects-export-objects[Saved Objects] UI in Kibana (*Stack Management* -> *Kibana* -> *Saved Objects*) to export and import any connectors used by your detection rule actions _before_ you export and import the detection rules.
+=================
+
+To export and import detection rules:
+
+. Go to *Detect* -> *Rules*.
 . To export rules:
 .. In the All rules table, select the rules you want to export.
-.. Select *Bulk actions* -> *Export selected*.
+.. Select *Bulk actions* -> *Export selected*, then save the exported file.
++
+NOTE: You cannot export Elastic prebuilt rules.
+. To import rules:
+.. Click *Import rules*.
+.. Drag and drop the file that contains the detection rules.
 +
-NOTE: You cannot export prebuilt rules.
+NOTE: Imported rules must be in an `.ndjson` file.
+.. Select *Overwrite existing detection rules with conflicting Rule ID* to update existing rules if they match any rules in the import file.
```

Comment:
This didn't make it obvious to me that existing rules' actions will get updated to match the imported rules' actions if both rules have the same criteria. Maybe "...conflicting Rule ID* if you want imported rules to overwrite the actions associated with existing rules when both rules have the same criteria."

Reply:
I added another sentence to make this clearer -- check out the latest draft and let me know if that works better. Thanks!